content/install/_index.md

```diff
@@ -24,6 +24,9 @@ if (macosPlatforms.indexOf(platform) !== -1) {
 <link rel="stylesheet" href="/css/install.css">
 
 # 1. Choose your platform
+
+Please click to highlight your platform; this will show the instructions for that platform.
+
```

Comment:
I feel that the new buttons are obvious enough that this sentence is no longer necessary.

Reply:
Agree!